### PR TITLE
[FBZ-10522] Create a README.md file for ey-cookbooks-stable-v7 repo

### PR DESCRIPTION
**Pull request by rupikakapoor**
### Description of your patch
Added README.md to ey-cookbook-stable-v7 

### Recommended Release Notes
Added README.md to ey-cookbook-stable-v7 

### Estimated risk
Low

###Components involved
none

### CODE REVIEW

File: EnvironmentVariables.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Configuration Via Environment Variables
+
+Many features in an Engine Yard Cloud stable-v6 environment can now be enabled
```
Comment (dvalfre) [inline EnvironmentVariables.md:3]: let's remember this is for v7 😄

File: EnvironmentVariables.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+and configured via [environment variables](https://support.cloud.engineyard.com/hc/en-us/articles/360007661794).
+
+This makes working with the Engine Yard Cloud platform much easier and 
+removes the need for custom Chef recipes in many cases.
+
+A list of supported settings is documented in the next section.
+If you are missing a setting, please request it by [opening a GitHub issue](https://github.com/engineyard/ey-cookbooks-stable-v7/issues/new).
+
+## Supported Settings
```
Comment (dvalfre) [inline EnvironmentVariables.md:12]: @rupikakapoor we are missing some settings here, particularly the ones about versions of PostgreSQL.
Also, existing ones like EY_SIDEKIQ_* are pointing to empty files.

File: EnvironmentVariables.md (hunk 1 above)
Comment (rupikakapoor) [reply EnvironmentVariables.md:3]: Corrected now

File: EnvironmentVariables.md (hunk 2 above)
Comment (rupikakapoor) [reply EnvironmentVariables.md:12]: yeah, this will be addresses in FBZ-10526

**Pull request merged**